WrightTools/kit/_array.py

```diff
@@ -433,3 +434,32 @@ def enforce_mask_shape(mask, shape):
     """
     red = tuple([i for i in range(len(shape)) if shape[i] == 1])
     return mask.max(axis=red, keepdims=True)
+
+
+def signed(d0):
+    """Tells bluesky whether to sign data. We have a 7.5 % tolerance right now.%
+
+    Parameters
+    -------------
+    d0: Input data array
+
+
+
+    Returns
+    -------
+    A boolean; True if the data is signed and False if the data is not.
+    """
+
+    maxd0 = np.max(d0)
+    mind0 = np.min(d0)
+
+    tolerance = (maxd0 - np.absolute(mind0)) / (maxd0 + np.absolute(mind0))
+
+    if tolerance < 7.5 * 10 ** (-2):
+        if min(d0) > 0:
+            return False
+        else:
+            return True
+
+    else:
+        return False
```

Comment:
```suggestion
    if tolerance >= 7.5e-2:
        return False
    else:
        return mind0 <= 0
```
More compact, perhaps more readable.  
The evaluation statements you have are actually booleans themselves (e.g. 1>=0 evaluates to True), so you don't have to turn them into True or False values--you can just return them.